src/test/FuzzerImpl.cpp

```diff
@@ -644,14 +644,17 @@ TransactionFuzzer::initialize()
                 offerParameters.emplace_back(
                     FuzzUtils::OfferParameters(bid, sell, pk, price, amount));
                 ++accountParameters[pk].mNumOffers;
-                LedgerEntry offerEntry;
-                offerEntry.data.type(OFFER);
-                offerEntry.data.offer().buying = bid;
-                offerEntry.data.offer().selling = sell;
-                offerEntry.data.offer().price = price;
-                offerEntry.data.offer().amount = amount;
-                accountParameters[pk].mOfferLiabilities +=
-                    getOfferBuyingLiabilities(header, offerEntry);
+                if (sell.type() == ASSET_TYPE_NATIVE)
+                {
+                    LedgerEntry offerEntry;
+                    offerEntry.data.type(OFFER);
+                    offerEntry.data.offer().buying = bid;
+                    offerEntry.data.offer().selling = sell;
+                    offerEntry.data.offer().price = price;
+                    offerEntry.data.offer().amount = amount;
+                    accountParameters[pk].mNativeOfferLiabilities +=
+                        getOfferBuyingLiabilities(header, offerEntry);
```

Comment:
I think you want selling liabilities? The buying liabilities relate to the amount you would receive if all of the offers were crossed simultaneously. Think about it this way, if you are selling native then you are _not_ buying native so once again these all have different units.

Reply:
Indeed!  Fixed.